src/popup/router/components/BalanceInfo.vue

```diff
@@ -31,11 +32,11 @@
 
 <script>
 import { mapGetters } from 'vuex';
-import DropdownArrow from '../../../icons/dropdownarrow.svg';
+import expandedAngleArrow from '../../../icons/expanded-angle-arrow.svg';
 
 export default {
   components: {
-    DropdownArrow,
+    'expanded-angle-arrow': expandedAngleArrow,
```

Comment:
`expandedAngleArrow` is also a component, so I propose to keep it as before:
```suggestion
    ExpandedAngleArrow,
```